python/tvm/auto_scheduler/measure.py

```diff
@@ -566,9 +592,13 @@ def timed_func(inp, build_res):
 
         if error_no == 0:
             try:
-                # TODO(FrozenGene): Update to ndarray.non-empty.
                 args = [ndarray.empty(get_const_tuple(x.shape), x.dtype, ctx) for x in
                         build_res.args]
+                assert tvm.get_global_func("tvm.contrib.random.random_fill", True), \
+                    "Do you enable USE_RANDOM in the config.cmake?"
+                random_fill = tvm.get_global_func("tvm.contrib.random.random_fill")
```

Comment:
```suggestion
                random_fill = tvm.get_global_func("tvm.contrib.random.random_fill", True)
                assert random_fill, "Please set USE_RANDOM to On the config.cmake."
```
We should call tvm.get_global_func only once.

Reply:
done.